docs/caching.md

```diff
@@ -0,0 +1,26 @@
+# Caching
+
+### Akamai
+
+We use Akamai, a content delivery network, (https://www.akamai.com/), to cache the entirety of consumerfinance.gov (but not our development servers). We invalidate any given page in Wagtail when it is published or unpublished (by hooking up the custom class `AkamaiBackend` (in https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/akamai_backend.py) to Wagtail's frontend cache invalidator (http://docs.wagtail.io/en/v1.9/reference/contrib/frontendcache.html). By default, we clear the Akamai cache any time we deploy. 
+
+There are certain pages that do not live in Wagtail or are impacted by changes on another page (imagine our newsroom page that lists titles of other pages - https://www.consumerfinance.gov/about-us/newsroom/) or another process (imagine data from Socrata gets updated) and thus will display outdated content until the page's time to live (TTL) has expired, a deploy has happened, or if someone manually invalidates that page. Our default TTL is 24 hours.
+
+### Django caching
+
+Starting in December 2017, we use template fragment caching (https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/fragment_cache_extension.py) to cache all or part of a template.  It is enabled on our "post previews", snippets of a page that we display on results of filterable pages (e.g. http://consumerfinance.gov/about-us/blog, https://www.consumerfinance.gov/data-research/research-reports/). 
+
+It can easily be enabled on other templates. See https://github.com/cfpb/cfgov-refresh/pull/3663/files as an example of the code that would need to be introduced to cache a new fragment. 
```

Comment:
TRIVIAL: Similar to above we could hide the actual URL behind a link to shorten the text:
```
See [this PR](https://github.com/cfpb/cfgov-refresh/pull/3663/files) as an example of the code…
```

Reply:
Hmmm yeah I could do that, but what's the value of obscuring the URL?